resources/js/Pages/Home.vue

```diff
@@ -42,18 +43,51 @@
             user: Object,
         },
         methods: {
+            checkEmpty() {
+                if( !this.form.itemsInput ) {
+                    this.error = {
+                        type: 'warning',
+                        message: 'item-form-error-message-empty'
+                    };
+                    return false;
```

Comment:
Nice work! I think, since we are already mutating the internal state of the component, perhaps it is not required to return an explicit boolean? We can just perform the checks and at the end (before sending) check whether the error is set or not.